dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/pages/security/WarningInstancePage.java

```diff
@@ -0,0 +1,156 @@
+
+    @FindBy(className = "items")
+    private List<WebElement> warningInstanceList;
+
+    @FindBys({
+        @FindBy(className = "el-popconfirm"),
+        @FindBy(className = "el-button--primary"),
+    })
+    private List<WebElement> buttonConfirm;
+
+    private final WarningInstanceForm createWarningInstanceForm = new WarningInstanceForm();
+    private final WarningInstanceForm editWarningInstanceForm = new WarningInstanceForm();
+    private final WarningInstanceForm alertPluginForm = new WarningInstanceForm();
+
+
+    public WarningInstancePage(RemoteWebDriver driver) {
+        super(driver);
+    }
+
+    public WarningInstancePage create(String alarmInstanceName, String alarmPluginName, String inputPluginContent, String keyword) {
+        buttonCreateWarningInstance().click();
+
+        createWarningInstanceForm().inputAlarmInstanceName().sendKeys(alarmInstanceName);
+        createWarningInstanceForm().selectAlarmPlugin().click();
+        SelectAlarmPlugin(alarmPluginName);
+        createWarningInstanceForm().inputWebhook().sendKeys(inputPluginContent);
+        createWarningInstanceForm().inputKeyword().sendKeys(keyword);
+        createWarningInstanceForm().radioNoEnableProxy().click();
+        createWarningInstanceForm().buttonSubmit().click();
+        return this;
+    }
+
+
+    public WarningInstancePage update(String warningInstanceName, String editWarningInstanceName) {
+        warningInstanceList()
+            .stream()
+            .filter(it -> it.findElement(By.className("name")).getAttribute("innerHTML").contains(warningInstanceName))
+            .flatMap(it -> it.findElements(By.className("edit")).stream())
+            .filter(WebElement::isDisplayed)
+            .findFirst()
+            .orElseThrow(() -> new RuntimeException("No edit button in alarmInstance list"))
+            .click();
+
+        editWarningInstanceForm().inputAlarmInstanceName().clear();
+        editWarningInstanceForm().inputAlarmInstanceName().sendKeys(editWarningInstanceName);
+        editWarningInstanceForm().buttonSubmit().click();
+
+        return this;
+    }
+
+    public WarningInstancePage delete(String WarningInstance) {
+        warningInstanceList()
+            .stream()
+            .filter(it -> it.findElement(By.className("name")).getAttribute("innerHTML").contains(WarningInstance))
+            .flatMap(it -> it.findElements(By.className("delete")).stream())
+            .filter(WebElement::isDisplayed)
+            .findFirst()
+            .orElseThrow(() -> new RuntimeException("No delete button in alarmInstance list"))
+            .click();
+
+        buttonConfirm()
+            .stream()
+            .filter(WebElement::isDisplayed)
+            .findFirst()
+            .orElseThrow(() -> new RuntimeException("No confirm button when deleting"))
+            .click();
+
+        return this;
+    }
+
+    public WarningInstancePage SelectAlarmPlugin(String alarmPluginName) {
```

Comment:
method names are usually camel-case with first letter lowercase 

```suggestion
    public WarningInstancePage selectAlarmPlugin(String alarmPluginName) {
```